base_layer/core/src/proof_of_work/target_difficulty.rs

```diff
@@ -24,21 +24,23 @@ use crate::{
     consensus::ConsensusManagerError,
     proof_of_work::{difficulty::DifficultyAdjustment, lwma_diff::LinearWeightedMovingAverage, Difficulty},
 };
+use std::cmp;
 use tari_crypto::tari_utilities::epoch_time::EpochTime;
 
 /// Returns the estimated target difficulty for the provided set of target difficulties.
 pub fn get_target_difficulty(
     target_difficulties: Vec<(EpochTime, Difficulty)>,
     block_window: usize,
     target_time: u64,
-    initial_difficulty: Difficulty,
+    min_difficulty: Difficulty,
     max_block_time: u64,
 ) -> Result<Difficulty, ConsensusManagerError>
 {
-    let mut lwma = LinearWeightedMovingAverage::new(block_window, target_time, initial_difficulty, max_block_time);
+    let mut lwma = LinearWeightedMovingAverage::new(block_window, target_time, min_difficulty, max_block_time);
     for (epoch, difficulty) in target_difficulties {
         lwma.add(epoch, difficulty)?;
     }
-    let target_difficulty = lwma.get_difficulty();
+    let mut target_difficulty = lwma.get_difficulty();
+    target_difficulty = cmp::max(target_difficulty, min_difficulty);
```

Comment:
I dont think this might be an issue, but we should just check that as of now, blake has not dipped below its min

Reply:
Good point. I think there are cases where it is below the target. Will try sync a new chain and see how it does